tests/e2e/cucumber/environment/index.ts

```diff
@@ -145,9 +154,35 @@ AfterAll(async () => {
   if (state.browser) {
     await state.browser.close()
   }
+
+  if (config.reportTracing) {
+    // move failed tracing reports
+    const failedDir = path.dirname(config.tracingReportDir) + '/failed'
```

Comment:
should we change path here?
https://github.com/owncloud/web/pull/11571/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eR1512-R1513

to `reports/e2e/playwright/tracing/failed`

Reply:
All failed tracings will be moved to the original path after all tests are executed

Reply:
So that means there will be no change in the tracing paths at the end